hybrasyl/Objects/User.cs

```diff
@@ -76,6 +68,12 @@ public static string GetStorageKey(string name)
 
         public string StorageKey => string.Concat(GetType().Name, ':', Name.ToLower());
 
+        [JsonProperty]
+        public string Identifier { get; private set; }

```

Comment:
what do you think about potentially making this `Uuid` and `AccountUuid` and making that change everywhere?

something about `Identifier`....idk, maybe it's nothing but I feel like i'd rather be explicit

Reply:
🤷